devsetup/Makefile

```diff
@@ -97,11 +98,27 @@ edpm_compute_repos: ## Configure EDPM compute VM with repositories
 	$(eval $(call vars))
 	scripts/edpm-compute-repos.sh ${EDPM_COMPUTE_SUFFIX}
 
+.PHONY: edpm_nodes
```

Comment:
I think this target could probably just take the place of edpm_compute/edpm_compute_cleanup. No reason to have multiple ways of doing it. If you just want one node, you'd set EDPM_TOTAL_NODES=1.